src/platforms/common/blackpill-f4/README.md

```diff
@@ -67,12 +78,24 @@ make PROBE_HOST=blackpill-f4x1cx SHIELD=1
   - Release BOOT0
 - Upload the firmware:
 ```
-./dfu-util -a 0 --dfuse-address 0x08000000:leave -R -D blackmagic.bin
+./dfu-util -d 0483:df11 --alt 0 -s 0x08000000:leave -D build_blackpill_xxxxxx/blackmagic_blackpill_xxxxxx_firmware.bin
```

Comment:
The load address here should be `0x08004000` should it not?

Reply:
Yes :)